data/templates/Pipfile

```diff
@@ -0,0 +1,14 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+${dependencies}
+
+[dev-packages]
+mock = "*"
+
+[requires]
```

Comment:
My sense is that we should remove this, since we support both python2 and 3 for (almost) all projects at present. What do you think? Create multiple pipfiles instead? https://github.com/pypa/pipenv/issues/1050 suggests that multiple python versions is a bit controversial...

Reply:
ack